common/app/model/content/Atoms.scala

```diff
@@ -553,12 +586,26 @@ object TimelineAtom {
     atom.data.asInstanceOf[AtomData.Timeline].timeline,
     events = atom.data.asInstanceOf[AtomData.Timeline].timeline.events map TimelineItem.make _
   )
+
+  def renderFormattedDate(item: TimelineItem, format: String): String = {
+    item.toDate match {
+      case Some(date) => {
+        val beginRange = DateTimeFormat.forPattern(format).print(item.date)
+        val endRange = DateTimeFormat.forPattern(format).print(date)
+        s"$beginRange&ndash;$endRange"
+      }
+      case None => DateTimeFormat.forPattern(format).print(item.date)
+    }
+  }
+
```

Comment:
Would writting unit test for this function help making sure the implemenation is right and more importantly isn't changed by accident?